test/scylla_gdb/test_misc.py

```diff
@@ -187,6 +187,11 @@ def coro_task(gdb, scylla_gdb):
         name = scylla_gdb.resolve(vtable_addr)
         if name and '.resume' in name.strip():
             gdb.write(f'{name}\n')
+    # This test fails sometimes, but rarely and unreliably.
+    # We want to get a coredump from it the next time it fails.
+    # Sending a SIGSEGV should induce that.
+    # https://github.com/scylladb/scylladb/issues/22501
+    gdb.execute("signal SIGSEGV")
```

Comment:
Do you want a coredump of scylla, or one for gdb?

Reply:
For Scylla, of course. The suspicion is that `scylla-gdb.py` (specifically: `find_vptrs`) is misbehaving sometimes. I want to run it, with the same Scylla state which confuses it, to debug it.